src/SvgIcon/SvgIcon.js

```diff
@@ -0,0 +1,72 @@
+// @flow weak
+
+import React, { PropTypes } from 'react';
+import { createStyleSheet } from 'jss-theme-reactor';
+import classNames from 'classnames';
+import { easing } from '../styles/transitions';
+
+export const styleSheet = createStyleSheet('SvgIcon', (theme) => {
+  return {
+    SvgIcon: {
```

Comment:
What do you think about this convention https://github.com/airbnb/javascript/tree/master/css-in-javascript#naming? I believe that we are using camelCase everywhere else.